nucliadb_node/Makefile

```diff
@@ -25,3 +25,8 @@ test:
 .PHONY: test-cov
 test-cov:
 	pytest -rfE --cov=nucliadb_node --cov-config=../.coveragerc -s --tb=native -v --cov-report term-missing:skip-covered --cov-report xml nucliadb_node
+
+.PHONY: build
+build:
+	export RUSTFLAGS="--cfg=prometheus_metrics"
```

Comment:
`export` doesn't take effect here, as commands are executed in different subshells. Use target-specific variables instead to make this env var local to the target:
https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html

```make
build: export RUSTFLAGS="--cfg=prometheus_metrics"
build:
        cargo build
```